src/app/view/components/_common/dot-push-publish-dialog/dot-push-publish-dialog.component.ts

```diff
@@ -53,18 +53,21 @@ export class DotPushPublishDialogComponent implements OnInit, OnDestroy {
     ) {}
 
     ngOnInit() {
-        this.loadMessagesAndFilters();
         this.dotPushPublishDialogService.showDialog$
             .pipe(takeUntil(this.destroy$))
             .subscribe((data: DotPushPublishEvent) => {
-                this.eventData = data;
-                this.assetIdentifier = this.eventData.assetIdentifier;
-                this.pushActions = this.getPushPublishActions(this.i18nMessages);
-                this.initForm({
-                    filterKey: this.defaultFilterKey
-                });
-                this.setDialogConfig(this.i18nMessages, this.form);
-                this.dialogShow = true;
+                this.loadMessagesAndFilters()
```

Comment:
I think this if should go here:

https://github.com/dotCMS/core-web/pull/1306/files#diff-8d95ff3168710d370ed7f360881e6f4dR123

Reply:
done